artpaint/application/PixelOperations.h

```diff
@@ -295,8 +211,6 @@ inline uint32 src_over_fixed(uint32 dst, uint32 src)
 	if (result_alpha == 0)
 		return 0;
 
-	// r-rgb * r-a = s-rgb * s-a + d-rgb * d-a * (1 - s-a)
-
```

Comment:
oh sorry, can you please undo these?  These aren't commented-out code, they're describing the blending operation via pseudo-code.  I guess I should add a line explaining that.